src/sqlfluff/core/default_config.cfg

```diff
@@ -123,17 +129,12 @@ spacing_within = touch:inline
 spacing_after = touch:inline
 
 [sqlfluff:layout:type:function_name]
-spacing_after = touch:inline
-
-[sqlfluff:layout:type:select_except_clause]
-# This prevents space between EXCEPT following brackets:
-# e.g. SELECT * EXCEPT()
 spacing_within = touch:inline
+spacing_after = touch:inline
```

Comment:
I prefer this and think it should be the default as that's what the BigQuery docs have, but it means we need to update `test/fixtures/linter/autofix/bigquery/006_fix_ignore_templating/after.sql` (and probably before.sql too).

Reply:
yep - updated 👍